src/lifx/light.js

```diff
@@ -510,9 +510,9 @@ Light.prototype.colorZones = function(startIndex, endIndex, hue, saturation, bri
   validate.optionalCallback(callback, 'color zones method');
 
   // Convert HSB values to packet format
-  hue = Math.round(hue / constants.HSBK_MAXIMUM_HUE * 65535);
-  saturation = Math.round(saturation / constants.HSBK_MAXIMUM_SATURATION * 65535);
-  brightness = Math.round(brightness / constants.HSBK_MAXIMUM_BRIGHTNESS * 65535);
+  hue = Math.round((hue / constants.HSBK_MAXIMUM_HUE) * 65535);
+  saturation = Math.round((saturation / constants.HSBK_MAXIMUM_SATURATION) * 65535);
+  brightness = Math.round((brightness / constants.HSBK_MAXIMUM_BRIGHTNESS) * 65535);
```

Comment:
Revert these unrelated changes.

Reply:
I revert this changes, but it drives me crazy.... infact nobody normal knows the order of execution of the operators * /.